contracts/test/MapleGlobals.t.sol

```diff
@@ -1,273 +1,439 @@
+        assertEq(globals.fundingPeriod(), 1 days);
+    }
+    
+    function test_setSwapOutRequired() public {
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
+
+        assertTrue(!fakeGov.try_mapleGlobals_setSwapOutRequired(address(globals), 10_000));
+        assertTrue( realGov.try_mapleGlobals_setSwapOutRequired(address(globals), 10_000));
+        assertEq(globals.swapOutRequired(), 10_000);
+    }
+    
+    function test_setPriceOracle() public {
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
+
+        assertEq(globals.oracleFor(address(1)), address(0));
+        
+        assertTrue(!fakeGov.try_mapleGlobals_setPriceOracle(address(globals), address(1), address(2)));
+        assertTrue( realGov.try_mapleGlobals_setPriceOracle(address(globals), address(1), address(2)));
+
+        assertEq(globals.oracleFor(address(1)), address(2));
+    }
+    
+    function test_setPendingGovernor() public {
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
+
+        assertEq(globals.pendingGovernor(), address(0));
+        
+        assertTrue(!fakeGov.try_mapleGlobals_setPendingGovernor(address(globals), address(1)));
+        assertTrue( realGov.try_mapleGlobals_setPendingGovernor(address(globals), address(1)));
+
+        assertEq(globals.pendingGovernor(), address(1));
+    }
+
+    /***************/
+    /*** Getters ***/
+    /***************/
+    function test_getLatestPrice() public {
+        Governor     realGov = new Governor();
+        MapleGlobals globals = new MapleGlobals(address(realGov), address(1), address(2));
+
+        address oracleMock = address(new OracleMock());
+        realGov.mapleGlobals_setPriceOracle(address(globals), address(1), oracleMock);
+        assertEq(globals.getLatestPrice(address(1)), 100);
+    }
+
+    function test_isValidSubFactory() public {
+        Governor     realGov = new Governor();
+        MapleGlobals globals = new MapleGlobals(address(realGov), address(1), address(2));
+
+        address loanFactoryMock = address(1);
+        address poolFactoryMock = address(2);
+        address subFactoryMock  = address(new SubFactoryMock(1));
+
+        realGov.mapleGlobals_setValidLoanFactory(address(globals), loanFactoryMock, true);
+        realGov.mapleGlobals_setValidLoanFactory(address(globals), poolFactoryMock, true);
+
+        assertTrue(!globals.isValidSubFactory(loanFactoryMock, subFactoryMock, 1));
+
+        realGov.mapleGlobals_setValidSubFactory(address(globals), loanFactoryMock, subFactoryMock, true);
+
+        assertTrue(!globals.isValidSubFactory(poolFactoryMock, subFactoryMock, 1));  // Wrong superfactory
+        assertTrue(!globals.isValidSubFactory(loanFactoryMock, subFactoryMock, 2));  // Wrong subfactory type
+        assertTrue( globals.isValidSubFactory(loanFactoryMock, subFactoryMock, 1));  // Wrong subfactory type
+    }
```

Comment:
`makeGlobalsAndGovs` not used again. Perhaps throw this in the other contract and `MapleGlobalsGeneralTests`.